packages/api-aco/__tests__/graphql/record.gql.ts

```diff
@@ -32,7 +32,7 @@ export const CREATE_RECORD = /* GraphQL */ `
     mutation CreateRecord($data: SearchRecordCreateInput!) {
         search {
             createRecord(data: $data) {
-                data ${DATA_FIELD("id")}
+                data ${DATA_FIELD()}
```

Comment:
Several things about this `DATA_FIELD()`: 
- you never use the `extra` parameter
- you can convert this into a const value (not a function), and remove the curly braces from it, then the `extra` thing can be added where necessary, without passing any arguments, like this:

```
const DATA_FIELD = /* GraphQL */ `
    id
    type
`;

export const LIST_RECORDS = /* GraphQL */ `
    query ListRecords($where: SearchRecordListWhereInput, $search: String, $limit: Int, $after: String, $sort: [AcoListSort]) {
        search {
            listRecords(where: $where, search: $search, limit: $limit, after: $after, sort: $sort) {
                data {
                    ${DATA_FIELD}
                    extraField                 # <---- extra fields
                }
                meta ${LIST_META_FIELD}
                error ${ERROR_FIELD}
            }
        }
    }
`;
```